src/AWS.Deploy.Orchestration/DeploymentCommands/BeanstalkEnvironmentDeploymentCommand.cs

```diff
@@ -3,19 +3,66 @@
+
+            var deploymentPackage = recommendation.DeploymentBundle.DotnetPublishZipPath;
+            var environmentName = cloudApplication.Name;
+            var applicationName = (await orchestrator._awsResourceQueryer.ListOfElasticBeanstalkEnvironments())
+                .Where(x => string.Equals(x.EnvironmentId, cloudApplication.UniqueIdentifier))
+                .FirstOrDefault()?
+                .ApplicationName;
+
+            if (string.IsNullOrEmpty(applicationName))
+            {
+                var message = $"Could not find any Elastic Beanstalk application that contains the following environment: {environmentName}";
+                throw new AWSResourceNotFoundException(DeployToolErrorCode.FailedToFindElasticBeanstalkApplication, message);
+            }
+
+            orchestrator._interactiveService.LogMessageLine($"Initiating deployment to {cloudApplication.DisplayName}...");
+
+            var versionLabel = $"v-{DateTime.Now.Ticks}";
+            var s3location = await orchestrator._elasticBeanstalkHandler.CreateApplicationStorageLocationAsync(applicationName, versionLabel, deploymentPackage);
+            await orchestrator._s3Handler.UploadToS3Async(s3location.S3Bucket, s3location.S3Key, deploymentPackage);
+            await orchestrator._elasticBeanstalkHandler.CreateApplicationVersionAsync(applicationName, versionLabel, s3location);
+            var additionalSettings = orchestrator._elasticBeanstalkHandler.GetAdditionalSettings(recommendation);
+
+            var success = await orchestrator._elasticBeanstalkHandler.UpdateEnvironmentAsync(applicationName, environmentName, versionLabel, additionalSettings);
+
+            if (success)
+            {
+                orchestrator._interactiveService.LogMessageLine($"The Elastic Beanstalk Environment {environmentName} has been successfully updated to the application version {versionLabel}" + Environment.NewLine);
+            } 
+            else
+            {
+                throw new ElasticBeanstalkException(DeployToolErrorCode.FailedToUpdateElasticBeanstalkEnvironment, "Failed to update the Elastic Beanstalk environment");
+            }
+        }
+        public async Task<List<DisplayedResourceItem>> GetDeploymentOutputsAsync(IDisplayedResourcesHandler displayedResourcesHandler, CloudApplication cloudApplication, Recommendation recommendation)
+        {
+            var displayedResources = new List<DisplayedResourceItem>();
+            var environment = await displayedResourcesHandler.AwsResourceQueryer.DescribeElasticBeanstalkEnvironment(cloudApplication.Name);
+            var data = new Dictionary<string, string>() {{ "Endpoint", $"http://{environment.CNAME}/" }};
+            var description = "An AWS Elastic Beanstalk environment is a collection of AWS resources running an application version.";
+            displayedResources.Add(new DisplayedResourceItem(environment.EnvironmentName, description, "Elastic Beanstalk Environment", data));
```

Comment:
Make the type parameter a constant. We should also sync with Chris to see if they need more information for the toolkit.